frontend/src/Pages/FC/Badges.js

```diff
@@ -259,6 +260,7 @@ const View = () => {
       row.character.name.toLowerCase().includes(filters?.name.toLowerCase())
   );
 
+  useAppendTitle("Issue Badges");
```

Comment:
I would call this page `Specialist Badges` or `Badges` so it is consistent with `Announcements`, `Bans` and `Commanders`

Reply:
`Badges` is already taken in the Guides section. `Specialist Badges` was a bit too long with the initial title scheme, but it will fit with your suggestion for shortening titles.